modules/csharp/examples/code_snippets/Program.cs

```diff
@@ -1674,36 +1674,54 @@ public Document Resolve(Conflict conflict)
 
 // N1QL QUERY
 
-
-  public  List<Result> testN1QLQueryString (Database argDB)
+  public List<Result> docsonly_N1QLQueryString(Database argDB)
   {
-      // For Documentation
+      var doc_type = "hotel";
+
       DatabaseConfiguration dbCfg = new DatabaseConfiguration();
       dbCfg.Directory =
           Path.Combine(Service.GetInstance<IDefaultDirectoryResolver>().DefaultDirectory(), username);
```

Comment:
I think this is redundant. The default value of Directory property is already defined.